lib/components/viewers/stop-viewer.js

```diff
@@ -292,11 +306,21 @@ class StopViewer extends Component {
 
     let timezoneWarning
     if (!inHomeTimezone) {
-      const timezoneCode = format(Date.now(), 'z', {
-        // To avoid ambiguities for now, use the English-US timezone abbreviations ("EST", "PDT", etc.)
-        locale: dateFnsUSLocale,
-        timeZone: homeTimezone
-      })
+      // In schedule view, the time zone code should be that of the entered date,
+      // or the current day in the live view.
+      // This is to account for daylight time changes, especially when the live and
+      // schedule views are in different daylight saving periods.
+      const timezoneCode = formatTz(
+        isShowingSchedule && date
+          ? parse(date, inputDateFormat, new Date())
+          : new Date(), // TODO: mock for percy tests,
+        'z',
+        {
```

Comment:
This is causing the timezone code to change & update in the alert if the year starts with 0, which is a little jarring as you're typing.
<img width="441" alt="image" src="https://github.com/opentripplanner/otp-react-redux/assets/115499534/0b91c440-f543-4655-87a4-69f2a45bcfa9">

Reply:
Good catch! Fixed in https://github.com/opentripplanner/otp-react-redux/pull/1021/commits/4d1e6153d292e5f4458d7e873458fe3afd3442f2.